examples/function-templates/javascript/INSTRUCTIONS.md

```diff
@@ -13,9 +13,9 @@ You will need to set the following environment variables as listed below:
 It as simple using the CLI command `npm run upload-ci`. This will perform two actions: upload the code, linking it to the app, and then finally activating the code ready for usage in both 
 
 ### Assigning an app to a field
-Delivery functions are meant to help with resolving field data, meaning the app has to be assigned to a field location. You can read more about different app locations [here](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/). 
+Functions are meant to help with resolving field data, meaning the app has to be assigned to a field location. You can read more about different app locations [here](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/). 
```

Comment:
i don't understand these DF readme instruction changes... you HAVE to be specific about specific function context instructions at some point and renaming all of this here doesn't even make sense to me... ?

Reply:
```suggestion
Functions that support resolving field data on the GraphQL API have to be assigned to a field location. You can read more about different app locations [here](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/). 
```

Reply:
Perhaps this suggestion helps clarify my concerns/how we should frame this to developers?

Reply:
perfect, thanks!